### DISCUSSION TITLE
Klarna shows test mode even when its enabled in Stripe

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#350] [Help] [answered]
Hi, Klarna shows test mode even when its enabled in Stripe. Please note we have enabled Live mode in Stripe.

**Comment by SusannaSwell** [comment, marked as answer]
Hello @virajsoni06, I was able to reproduce the issue and can confirm the bug. As updates on a fix become available, we will let you know.

**Reply by logeshswell** [reply]
As a temporary workaround, can you update the `mode` following this endpoint?
<img width="434" alt="Screenshot 2023-08-17 at 5 34 34 PM" src="https://github.com/swellstores/community/assets/100766131/4604b844-f6f2-4920-b85e-13bffaefdace">

**Reply by virajsoni06** [reply]
Thank you

**Comment by steven-k1** [comment]
@virajsoni06 This issue should be fixed now
